sdk/openfeature-nodejs-provider/README.md

```diff
@@ -0,0 +1,110 @@
+# OpenFeature DevCycle NodeJS Provider
+
+This library provides a NodeJS implementation of the OpenFeature Provider interface for DevCycle.
+
+## Building
+
+Run `nx build sdk-openfeature-nodejs-provider` to build the library.
+
+## Running Unit Tests
+
+Run `nx test sdk-openfeature-nodejs-provider` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Example App
+
+See the [example app](/examples/openfeature-nodejs) for a working example of the OpenFeature DevCycle NodeJS Provider.
+
+## Usage
+
+See our [documentation](https://docs.devcycle.com/sdk/server-side-sdks/node/) for more information.
+
+```typescript
+import { OpenFeature, Client } from '@openfeature/js-sdk'
+import { DevCycleProvider } from '@devcycle/openfeature-nodejs-provider'
+import { initialize } from '@devcycle/nodejs-server-sdk'
+
+... 
+
+// Initialize the DevCycle SDK
+const dvcClient = await initialize(DVC_SERVER_SDK_KEY).onClientInitialized()
+// Set the initialzed DevCycle client as the provider for OpenFeature
+OpenFeature.setProvider(new DevCycleProvider(dvcClient))
+// Get the OpenFeature client
+openFeatureClient = OpenFeature.getClient()
+// Set the context for the OpenFeature client, you can use 'targetingKey' or 'user_id'
+openFeatureClient.setContext({ targetingKey: 'node_sdk_test' })
```

Comment:
Setting the `targetingKey` on the client will apply it to every flag evaluation, effectively making this a user-scoped client. That's perfectly fine to do as long as that was the behavior you were expecting.

Reply:
Functionally it's the same thing to the provider if it's set on the `OpenFeature` global object or the client? I assume the contexts are merged and sent to the provider in the `resolve<Type>Evaluation` methods?

Reply:
Correct, but it can also be set at the invocation level. The context merging behavior is defined here.

https://openfeature.dev/specification/sections/evaluation-context#requirement-322